packages/components/src/components/Text/index.ts

```diff
@@ -7,7 +7,20 @@ export const TextComponentDefinition: ComponentDefinition = {
   id: 'text',
   name: 'Text',
   category: 'Contentful',
-  builtInStyles: ['cfMargin', 'cfPadding'],
+  builtInStyles: [
+    'cfMargin',
+    'cfPadding',
+    'cfFontSize',
+    'cfFontWeight',
```

Comment:
This is more of a product comment. I think to start, maybe it's useful to have `FontWeight` and `TextBold` and see how it plays with the usability of our text component, but I could also see it being redundant.

Reply:
Yep, the `cfTextBold` boolean will map to a "quick toggle" bold button while the `cfFontWeight` string will map to a select or input that has more options for various weights.

Both fields will be hooked up to the css styles generator for `font-weight` style, so the UI will need to account for the logic when both options are enabled in a component (but I don't think it will be too bad)